### DISCUSSION TITLE
RStudio preview pane showing blank PDF after rendering

### DISCUSSION THREAD

**Discussion by wzbillings** [quarto-dev/quarto-cli#629] [Q&A] [answered]
Hi all, I'm using quarto 0.9.238 with the latest daily build of RStudio (2022.06.0 Build 295). When I render quarto (.qmd) to PDF, the PDF is rendered correctly and looks fine if I open the localhost link in my web browser or view the static file. But the preview pane in RStudio shows a completely blank document. The number of pages is correct, and when I try highlighting things in the pdf, it appears to have embedded text, but I cannot see the text in the document.

Does anyone know how I can debug this issue?

**Comment by cderv** [comment]
For reference, this has been posted in the community also: https://community.rstudio.com/t/how-to-debug-an-issue-with-the-quarto-pdf-viewer-pane/134272

**Comment by jjallaire** [comment]
I am not able to reproduce this on either Windows or OS X. I have a feeling this has something to do with QtWebKit (our underlying browser engine) text rendering. One thing to try is to see whether resizing the Viewer pane causes the text to paint. Another thing to try is changing the rendering engine options here to see if any of them make a difference:

![Screen Shot 2022-04-13 at 1 50 51 PM](https://user-images.githubusercontent.com/104391/163240689-44c28f64-b3d0-4943-8e53-3a001aab1350.png)

**Reply by wzbillings** [reply]
Resizing the pane doesn't work, but zooming in with the viewer DOES make the text show up--as I increase the zoom, text begins to appear from largest to small text. (Nothing shows up at 80% Zoom or below, title shows up at 90%, headings and author at 130% and body text at 170%.) The same behavior happens regardless of what the rendering engine option is set to. I am doing this on Windows 10.

If it's not reproducible, is there anything I should try uninstall/reinstalling to see if it fixes the issue?

**Comment by jjallaire** [comment, marked as answer]
It's definitely a graphics card/rendering issue. You can try playing with the GPU settings mentioned above to see if that clears it.

**Reply by wzbillings** [reply]
Ok, thank you!
